src/components/Result.js

```diff
@@ -1,12 +1,21 @@
 import './Result.css';
 import React from 'react';
+import PropTypes from 'prop-types';
 
-function Result() {
+// Text.propTypes = {
+//   show: PropTypes.node.isRequired,
+// };
+function Result({ show }) {
   return (
     <div className="Result-container">
-      <h1 className="result">0</h1>
+      <h1 className="result">{show.next || show.total}</h1>
```

Comment:
Well done in this milstone 👍🏼 , but I would suggest that you display the operator as well as it is confusing when a user click an operator button without seeing an iteraction